openproblems/tasks/spatial_decomposition/methods/baseline.py

```diff
@@ -0,0 +1,40 @@
+from ....tools.decorators import method
+from ....tools.utils import check_version
+from ..utils import split_sc_and_sp
+
+import numpy as np
+
+
+@method(
+    method_name="Random Proportions",
+    paper_name="Random Proportions (baseline)",
+    paper_url="https://openproblems.bio",
+    paper_year=2022,
+    code_url="https://github.com/openproblems-bio/openproblems",
+    is_baseline=True,
+)
+def random_proportions(adata, test=False):
```

Comment:
this function could add a `n_times` parameter to bootstrap random sampling times.

Reply:
Interesting idea, though it would be a little tricky because each method has to return a single result. We could, if we wanted, have multiple random methods with different random seeds, but since we're not explicitly designating methods as "good" or "bad" we would be taking the minimum performance, not the average, which might defeat the point of bootstrapping.

Reply:
right, we may take the minimum performance from a set of random seeds.

Reply:
agree!

Reply:
Wouldn't bootstrapping take the _average_ performance from a set of random seeds? In the limit, taking the minimum performance from a set of random seeds would give "perfectly bad" performance, which is much worse than random.

Reply:
I agree that taking the mean of an n_times random run would make more sense than the min. The question is a bit what the variation in random runs are for these baselines... how much do we need to protect against outliers in the baseline? 

In general, it would be prudent.... but it might also not really fit into the pipeline as it currently looks. The mean would need to be computed after metric runs... so we'd need intermediate results storage and then averaging. This is not something we have yet...

Reply:
Agreed. I suspect that with the sizes of datasets we have, we shouldn't be very sensitive to outliers.